docs/using/installing/index.rst

```diff
@@ -492,14 +492,92 @@ The ZFS ``pool`` variable should match the pool name created in the ZFS section.
    "control-service":
       "hostname": "${CONTROL_NODE}"
       "port": 4524
+
+``${CONTROL_NODE}`` should be replaced with the address of the control node.
+The optional ``port`` variable is the port on the control node to connect to.
+This value must agree with the configuration for the control service telling it on what port to listen.
+
+The file must also include a ``dataset`` item.
+This selects and configures a dataset backend.
+All nodes must be configured to use the same dataset backend.
+
+.. _openstack-dataset-backend:
+
+OpenStack Block Device Backend Configuration
+............................................
+
+The OpenStack backend uses Cinder volumes as the storage for datasets.
+This backend can be used with Flocker dataset agent nodes run by OpenStack Nova.
+The configuration item to use OpenStack should look like:
+
+.. code-block:: yaml
+
+   dataset:
+       backend: "openstack"
+       region: "<region slug; for example, LON>"
+       auth_plugin: "<authentication plugin>"
+       ...
+
+Make sure that the ``region`` specified matches the region where the Flocker nodes run.
+OpenStack must be able to attach volumes created in that region to your Flocker agent nodes.
+
+Other items are typically required but vary depending on the OpenStack authentication plugin selected
```

Comment:
This is not good enough; as it is I basically have no idea how to proceed after reading this.

"these plugins" - what's a plugin? plugin to whom? how do I add one? how can I find the names of the plugins? how do I find their requirements? do I need to write my own?

As it is reading this I'm not sure if Flocker runs against Rackspace, even, and have no idea how to proceed for other OpenStack environments.

This can be follow-up issue if you wish.